source/write-operations/insert-documents.txt

```diff
@@ -78,43 +71,45 @@ To learn more about document structure and rules, see the
 Insert One Document
 -------------------
 
-To add a single document to a MongoDB collection, call the ``insertOne()``
-method and pass the document you want to add. After you call the
-``insertOne()`` method, chain the ``subscribe()`` method and pass in a subscriber.
+To add a single document to a MongoDB collection, call the ``insertOne()`` method
+and pass the document you want to add. Then, pass the ``insertOne()`` method to the static
```

Comment:
Hey Ross,

the reactor note does not cover exactly what Mono is about; it says:

   This guide uses the Project Reactor library to implement ``Publisher`` and
   ``Subscriber`` instances. To learn more about the Project Reactor library and how
   to use Reactor, see `Getting Started
   <https://projectreactor.io/docs/core/release/reference/#getting-started>`__
   in the Reactor documentation.
   
   The reason why I added this information about Mono is because in a previous PR, I got feedback that despite having the Mono note in the page, the reviewer was still confused about what Mono was doing. Especially since people may just control-f the page or skim until they find the entry they want, I think it's still good to keep the information about Mono.